backend/tracim_backend/lib/utils/authorization.py

```diff
@@ -227,6 +228,12 @@ def check(self, tracim_context: TracimContext) -> bool:
         )
 
 
+class TranslationEnabled(AuthorizationChecker):
+    def check(self, tracim_context: TracimContext):
+        if not tracim_context.app_config.TRANSLATION_SERVICE__ENABLED:
+            raise PageNotFound()
```

Comment:
This would be better done by *not* adding the translation endpoints if the config is not enabled if this is possible (I think yes).

Reply:
This is possible but a bit complicated/not standard. We do do this normally using application.
To gain some time, it was not decided to not consider translation as an application as far i remember. Also, doing app doesn't solve all as we do have other issue in this case: how we do handle complex case like html-document and translation which are both app ?

Reply:
after discussion with @grignards and @buxx , it seems complicated to do due to hapic limitation, so we keep it.